components/MainLayout.js

```diff
@@ -15,6 +16,10 @@ export default function MainLayout ({ children }) {
     <div>
       <Head>
         <title>React Bangkok 2.0.0</title>
+        <link rel='apple-touch-icon' href={reactbkk} />
```

Comment:
Sorry, seems like it doesn’t work yet.

![image](https://cloud.githubusercontent.com/assets/193136/26207612/3fac66da-3c12-11e7-8a47-ee25500b8e5b.png)

Upon looking at the result, it looks like this:

```html
<link rel="apple-touch-icon" href="function reactbkk(props) {
  return _react2.default.createElement(&#x27;svg&#x27;, (0, _extends3.default)({
    width: &#x27;274&#x27;,
    height: &#x27;245&#x27;,
    viewBox: &#x27;0 0 274 245&#x27;,
    xmlns: &#x27;http://www.w3.org/2000/svg&#x27;,
    xmlnsXlink: &#x27;http://www.w3.org/1999/xlink&#x27;
```

Unlike webpack, we are using Next.js which does not use webpack in the server rendering side! For loading SVG files, we are using [babel-plugin-inline-react-svg](https://github.com/kesne/babel-plugin-inline-react-svg) which transforms SVG file imports to stateless functional components, which is what you see here.